apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/MulitpleDatabaseFirestoreFixture.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright 2022, Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Http;
+using Google.Cloud.ClientTesting;
+using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Firestore.IntegrationTests
+{
+    [CollectionDefinition(nameof(MulitpleFirestoreDbFixture))]
+    public class MulitpleFirestoreDbFixture : CloudProjectFixtureBase, ICollectionFixture<MulitpleFirestoreDbFixture>
+    {
+        private const string FirestoreURl = "https://firestore.googleapis.com/v1";
+        // Credentials scope, To view and manage your Google Cloud Datastore data.
+        private const string FirestoreCredentialScope = "https://www.googleapis.com/auth/datastore";
+        private const string ProjectEnvironmentVariable = "FIRESTORE_TEST_PROJECT";
+        private static readonly string s_credentialsPath = Environment.GetEnvironmentVariable("GOOGLE_APPLICATION_CREDENTIALS");
+        private static GoogleCredential s_googleCredential;
+        private static HttpClient s_httpClient;
+
+        static MulitpleFirestoreDbFixture()
+        {
+            // Initalize gcloud credentials to be used while REST Api calls. 
+            s_googleCredential = GoogleCredential.FromFile(s_credentialsPath);
+            if (s_googleCredential.IsCreateScopedRequired)
+            {
+                s_googleCredential = s_googleCredential.CreateScoped(new[]
+                {
+                    FirestoreCredentialScope
+                });
+            }
+            s_httpClient = new HttpClientFactory()
+                .CreateHttpClient(new CreateHttpClientArgs()
+               {
+                   ApplicationName = "Firestore integration tests",
+                   GZipEnabled = true,
+                   Initializers = { s_googleCredential },
+               });
+
+        }
+
+        public MulitpleFirestoreDbFixture() : base(ProjectEnvironmentVariable)
+        {
+        }
+
+        // creates new firestore database currently using rest API once the Admin Apis are available will be migrated to Grpc.

```

Comment:
This comment is a little unclear at the moment (and it starts with a lower case letter - tiny nit).

I would suggest making this a regular XML comment:

```csharp
// TODO: Use Google.Cloud.Firestore.Admin.V1 when that supports this operation.

/// <summary>
/// Creates a new Firestore database using the REST API.
/// </summary>
```